src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -123,6 +124,10 @@ private function doAsk(OutputInterface $output, Question $question)
             $ret = trim($this->autocomplete($output, $question, $inputStream, \is_array($autocomplete) ? $autocomplete : iterator_to_array($autocomplete, false)));
         }
 
+        if ($output instanceof ConsoleSectionOutput) {
+            $output->addContent($ret);
```

Comment:
How will this work with hidden input? There might not be any additional lines to clear in that case

Reply:
Answering questions still implies to hit `enter`, empty and hidden inputs result in a blank line on the screen. Just tried it, works as expected with hidden questions